### DISCUSSION TITLE
Python TextureSystem doesn't exist

### DISCUSSION THREAD

**Discussion by Doudou3D** [OpenImageIO/oiio#3150] [General]
Anyone have a suggestion to know why TextureSystem on python is not accessible ?

Traceback (most recent call last):
  File "d:\Dev\Libs\oiio\testsuite\python-texturesys\src\test_texture_sys.py", line 12, in <module>
    texture_sys = oiio.TextureSystem()
AttributeError: module 'OpenImageIO' has no attribute 'TextureSystem'

**Comment by lgritz** [comment]
Python bindings for TextureSystem were added to OIIO 2.3, but did not exist in previous releases.
